tests/e2e/not-monetized.spec.ts

```diff
@@ -1,133 +1,124 @@
 import { pathToFileURL } from 'node:url';
 import { test, expect } from './fixtures/connected';
-import type { Locator } from '@playwright/test';
 import { setupPlayground } from './helpers/common';
 
 const walletAddressUrl = process.env.TEST_WALLET_ADDRESS_URL;
 
-test.beforeEach(async ({ popup, page }) => {
-  await popup.reload();
-  await page.bringToFront();
-});
-
-test('shows not monetized on empty tabs', async ({
-  popup,
-  i18n,
-  persistentContext: context,
-}) => {
-  const warning = popup.getByTestId('not-monetized-message');
-  const msg = i18n.getMessage('notMonetized_text_newTab');
-  await expect(warning).toBeVisible();
-  await expect(warning).toHaveText(msg);
-
-  await context.newPage().then((page) => page.bringToFront());
-  await expect(warning).toBeVisible();
-  await expect(warning).toHaveText(msg);
-});
-
-test('shows not monetized on internal pages', async ({
-  page,
+// We make extensive use of test.step in this test to reuse the same browser
```

Comment:
State will be persisted between steps?

Reply:
Yes, reusing same browser, which contains the state.

Reply:
`test.step` is just for labeling blocks in a single test.